### PR TITLE
change 1 and 2

### PR DESCRIPTION
**Pull request by renan-huanca-coderoad-com**
implemented feature1

### CODE REVIEW

File: src/main/java/Example2.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public static void main(String args[]) {
 		JavaSparkContext sc = new JavaSparkContext(conf);
 		JavaRDD<String> numbers = sc.textFile("numbers.txt");
 
-		logger.info("first:" + numbers.first());
+		logger.info(">>>> first:" + numbers.first());
```
Comment (carlos-fernandez-seagull-com) [inline src/main/java/Example2.java:17]: it seems to be fine

**Review by carlos-fernandez-seagull-com — APPROVED** [review]
it seems to be fine

**Comment by carlos-fernandez-seagull-com** [comment]
it is fine

File: src/main/java/Example2.java (hunk 1 above)
Comment (CristianVertiz) [reply src/main/java/Example2.java:17]: but just seems :D

**Review by CristianVertiz — APPROVED** [review]

File: src/main/java/Example2.java (hunk 1 above)
Comment (alvaro-patty-tierconnect-com) [reply src/main/java/Example2.java:17]: Hola

**Pull request merged**